index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3167ce6dab.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+</head>
+
+<body>
+    <header>
+        <div class="header container">
+            <a href="/">
+                <p class="header-logo">My Logo</p>
+            </a>
+            <nav>
+                <i class="header-icon fa-solid fa-bars"></i>
+            </nav>
+        </div>
+    </header>
+
+    <section class="section-info">
+        <div class="background">
+            <div class="info-display">
+                <h1 class="info-heading container">I´m Maury <br> Glad to see you!</h1>
```

Comment:
- [ ] [OPTIONAL] the font size of the h1 is not as the one in the Figma design please try to implement the same font size 

- [ ] Your Design 
![image](https://user-images.githubusercontent.com/61790587/164772354-bc9f54e2-c174-4733-8fc6-2704cb32905e.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164772389-cfb32453-2ec0-4486-818b-afd870cd4e12.png)